core/events/ui_events.js

```diff
@@ -10,13 +10,50 @@
  */
 'use strict';
 
-goog.provide('Blockly.Events.OldUi');
+goog.provide('Blockly.Events.Ui');
+goog.provide('Blockly.Events.UiBase');
 
 goog.require('Blockly.Events');
-goog.require('Blockly.Events.Ui');
+goog.require('Blockly.Events.Abstract');
 goog.require('Blockly.registry');
 goog.require('Blockly.utils.object');
 
+/**
+ * Class for a UI event.
```

Comment:
assuming i understand these changes correctly, i might add 'base class for a ui event, use the more specific event type in core/events if possible' or some other text explaining. because if i understand correctly this isn't meant to be used directly (?)

and same below for the plain Ui event, maybe add that it's there for compatibility but you should use the new event classes

Reply:
I've update Jsdoc for both and added `@deprecated` annotation to `Events.Ui`

Reply:
Adding `@deprecated` is an issue for travis until I also fix zoom event that is still using old class. Since I want to handle that in a seperate PR, I'm just going to update the jsdoc here.